src/controller/routes.py

```diff
@@ -0,0 +1,19 @@
+from flask import abort, Blueprint, send_from_directory
+import os
+
+routes = Blueprint('routes', __name__)
```

Comment:
can we rename this blueprint and file as static_routes or serve_static

Reply:
Yeah

Reply:
Wait can we just use this file for all the routes, even the ones for review?

Reply:
the review route has to do auth, row locking, and like two levels of db query, I need to put it somewhere would be nice to split up the two route modules or else we can make a controller class in a separate file with a single route module

Reply:
Okay I'll split it up